src/module/so-inspired.js

```diff
@@ -23,6 +23,75 @@ Hooks.on("updateUser", (user) => {
   updateSheetForInspo(user);
   //updatePlayerListInspo();
 });
+
+Hooks.on("tidy5e-sheet.renderActorSheet", (app, element, data) => {
+  if (app.actor.type !== "character") {
+    return;
+  }
+
+  const html = $(element);
+
+  const inspirationArea = html.find(".inspiration");
+  inspirationArea.hide();
+
+  let currentInspiration;
+  const actorOwner = game.users.find(
+    (u) => u.character?.uuid === app.actor.uuid
+  );
+  if (actorOwner) {
+    currentInspiration = actorOwner.getFlag("so-inspired", "inspirationCount");
+
+    let newInspirationArea = `
+      <div
+        data-tidy-render-scheme="handlebars"
+        class="counter flexrow new-inspiration"
+        style="width: 200px; height: 25px; display: flex; margin: 10px 10px 10px 10px"
+      >
+        <h4>Inspiration</h4>
+        <div class="counter-value">
+          <button
+            type="button"
+            class="add-inspiration-btn"
+            style="height: 20px; width: 20px; line-height: 0px; padding-left: 2px"
+          >
+            <i
+              class="fa-solid fa-plus"
+              style="color: var(--t5e-primary-font-color)"
```

Comment:
This CSS variable reference makes So Inspired! work with Tidy's light and dark modes 💪💅